index.html

```diff
@@ -30,14 +30,14 @@ <h2>Седона - небольшой городок в Аризоне,<br>
       <section>
         <h3>Заинтересовались?</h3>
         <p>Укажите предполагаемые даты поездки, и мы покажем вам лучшие предложения гостиниц в Седоне</p>
-        <p>
-          <button type="button">Поиск гостиницы в Седоне</button>
-          <!-- Здесь будет форма
-          Дата заезда: <time></time><span></span><br>
-          Дата выезда: <time></time><span></span><br>
-          Взрослые: <span> </span> Дети: <span></span><br>
-          <button type="submit">Найти</button>-->
-        </p>
+          <form action="https://echo.htmlacademy.ru" method="GET">
+            <button type="button">Поиск гостиницы в Седоне</button>
+            <label>Дата заезда: <input type="date" name="date-enter" value="24 апреля 2017"></label><br>
+            <label>Дата выезда: <input type="date" name="date-leave" value="4 июля 2017"></label><br>
```

Comment:
```suggestion
            <label>Дата заезда: <input type="text" name="date-enter" value="24 апреля 2017"></label><br>
            <label>Дата выезда: <input type="text" name="date-leave" value="4 июля 2017"></label><br>
```
так, как есть элементы управления календарём, о чём речь ниже.

В [ТЗ](https://up.htmlacademy.ru/htmlcss/28/project/sedona#sedona-3-12) написано про эти элементы

![image](https://user-images.githubusercontent.com/17790771/82838185-13c2a480-9ed4-11ea-9a25-d976ec7f57f2.png)

Размечаются, как кнопки.